src/modules/esl-a11y-group/core/esl-a11y-group.ts

```diff
@@ -10,54 +10,43 @@ export type GroupTarget = 'next' | 'prev' | 'current';
 export class ESLA11yGroup extends ESLBaseElement {
   public static is = 'esl-a11y-group';
 
-  @attr({defaultValue: '::parent::child'}) public targets: string;
+  @attr({defaultValue: '::child'}) public targets: string;
   @boolAttr({}) public clickOnActive: boolean;
 
-  private _$targets?: HTMLElement[];
-
+  /**
+   * @returns {HTMLElement} parent element of this
+   */
+  public get $parent(): HTMLElement {
+    return this.parentElement as HTMLElement;
+  }
   /**
    * @returns {HTMLElement[]} targets of plugin
    */
   public get $targets(): HTMLElement[] {
-    return this._$targets as HTMLElement[] || [];
-  }
-  public set $targets($targets: HTMLElement[]) {
-    if (!$targets) return;
-    this.unbindEvents();
-    this._$targets = $targets;
-    this.bindEvents();
-  }
-
-  protected attributeChangedCallback(attrName: string, oldVal: string, newVal: string) {
-    if (!this.connected && oldVal === newVal) return;
-    if (attrName === 'targets') this.updateTargets();
+    return TraversingQuery.all(this.targets, this) as [];
```

Comment:
It will be better if root will be this.$parent

Reply:
Done, thx!